[2015-05-06 06:54:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:54:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:54:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:54:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:54:56] log.ERROR: exception 'ErrorException' with message 'Undefined offset: 1' in C:\xampp\htdocs\histogram\app\controllers\AutomateController.php:101 Stack trace: #0 C:\xampp\htdocs\histogram\app\controllers\AutomateController.php(101): Illuminate\Exception\Handler->handleError(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 101, Array) #1 [internal function]: AutomateController->{closure}(Object(Maatwebsite\Excel\Readers\LaravelExcelReader)) #2 C:\xampp\htdocs\histogram\vendor\maatwebsite\excel\src\Maatwebsite\Excel\Excel.php(122): call_user_func(Object(Closure), Object(Maatwebsite\Excel\Readers\LaravelExcelReader)) #3 C:\xampp\htdocs\histogram\app\controllers\AutomateController.php(105): Maatwebsite\Excel\Excel->load('C:\\xampp\\htdocs...', Object(Closure)) #4 [internal function]: AutomateController->postSave() #5 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array) #6 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postSave', Array) #7 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postSave', Array) #8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #9 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array) #10 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable() #11 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #12 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #13 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #14 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #15 {main} [] []
[2015-05-06 06:55:03] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:05] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:08] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:21] log.ERROR: exception 'ErrorException' with message 'Undefined offset: -1' in C:\xampp\htdocs\histogram\app\controllers\ManualController.php:109 Stack trace: #0 C:\xampp\htdocs\histogram\app\controllers\ManualController.php(109): Illuminate\Exception\Handler->handleError(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 109, Array) #1 [internal function]: ManualController->{closure}(Object(Maatwebsite\Excel\Readers\LaravelExcelReader)) #2 C:\xampp\htdocs\histogram\vendor\maatwebsite\excel\src\Maatwebsite\Excel\Excel.php(122): call_user_func(Object(Closure), Object(Maatwebsite\Excel\Readers\LaravelExcelReader)) #3 C:\xampp\htdocs\histogram\app\controllers\ManualController.php(117): Maatwebsite\Excel\Excel->load('C:\\xampp\\htdocs...', Object(Closure)) #4 [internal function]: ManualController->postSave() #5 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array) #6 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postSave', Array) #7 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postSave', Array) #8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #9 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array) #10 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable() #11 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #12 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #13 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #14 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #15 {main} [] []
[2015-05-06 06:55:31] log.ERROR: exception 'ErrorException' with message 'Undefined index: x' in C:\xampp\htdocs\histogram\app\storage\views\a4145cdd75e745c280a02ce308271d68:97 Stack trace: #0 C:\xampp\htdocs\histogram\app\storage\views\a4145cdd75e745c280a02ce308271d68(97): Illuminate\Exception\Handler->handleError(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 97, Array) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('C:\\xampp\\htdocs...') #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array) #4 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents() #5 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render() #6 C:\xampp\htdocs\histogram\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'postSave', Object(Illuminate\View\View)) #10 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postSave', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #12 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array) #13 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable() #14 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2015-05-06 06:55:36] log.ERROR: exception 'ErrorException' with message 'Undefined offset: -1' in C:\xampp\htdocs\histogram\app\controllers\ManualController.php:109 Stack trace: #0 C:\xampp\htdocs\histogram\app\controllers\ManualController.php(109): Illuminate\Exception\Handler->handleError(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 109, Array) #1 [internal function]: ManualController->{closure}(Object(Maatwebsite\Excel\Readers\LaravelExcelReader)) #2 C:\xampp\htdocs\histogram\vendor\maatwebsite\excel\src\Maatwebsite\Excel\Excel.php(122): call_user_func(Object(Closure), Object(Maatwebsite\Excel\Readers\LaravelExcelReader)) #3 C:\xampp\htdocs\histogram\app\controllers\ManualController.php(117): Maatwebsite\Excel\Excel->load('C:\\xampp\\htdocs...', Object(Closure)) #4 [internal function]: ManualController->postSave() #5 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array) #6 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postSave', Array) #7 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postSave', Array) #8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #9 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array) #10 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable() #11 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #12 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #13 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #14 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #15 {main} [] []
[2015-05-06 06:55:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:55:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2015-05-06 06:56:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php:1429 Stack trace: #0 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 C:\xampp\htdocs\histogram\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 C:\xampp\htdocs\histogram\public\index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
